static/styles/app_components.css

```diff
@@ -191,6 +191,11 @@
         }
     }
 
+    a:hover,
+    a:focus {
+        text-decoration-line: none;
+    }
```

Comment:
This commit is good, but its commit message has a too-long title.

Check out the Zulip commit message guidelines for more details: https://zulip.readthedocs.io/en/latest/contributing/version-control.html#commit-messages